blogs/data-mover/data-mover-1.2.0-overview.md

```diff
@@ -0,0 +1,134 @@
+
+<h2>OADP's Data Mover Roadmap</h2>
+<p dir="auto">The OADP team introduced the Data Mover feature in OADP-1.1.0 as a tech preview feature.  During the course of OADP-1.1 and OADP-1.2 development we have also been actively contributing to the upstream Velero project.  Our work in the upstream has focused on providing the foundation for the Data Mover feature directly in Velero.  We are pleased to see the upstream Velero project has officially adopted the feature and will release a <a href="https://github.com/vmware-tanzu/velero/wiki/1.12-Roadmap">built in Data Mover</a> in Velero 1.12. 
+The Data Mover feature in OADP-1.2 will remain in tech preview as we continue to contribute to and evalute Velero's built in Data Mover for our customers.
+<h2>Components</h2>
+
+<h3><a href="https://github.com/openshift/oadp-operator">OADP OPERATOR</a>:</h3>
+
+<p dir="auto">OADP is the OpenShift API for Data Protection operator. This open source operator sets up and installs Velero on the OpenShift platform, allowing users to backup and restore applications. We will be installing Velero alongside the CSI plugin (modified version).</p>
+
+<h3><a href="https://github.com/vmware-tanzu/velero-plugin-for-csi">CSI PLUGIN</a>:</h3>
+
+<p dir="auto">The collection of Velero plugins for snapshotting CSI backed PVCs using the CSI beta snapshot APIs.</p>
+
+<h3><a href="https://github.com/backube/volsync">VOLSYNC</a>:</h3>
+
+<p>VolSync is a Kubernetes operator that performs asynchronous replication of persistent volumes within, or across, clusters. The replication provided by VolSync is independent of the storage system. This allows replication to and from storage types that don’t normally support remote replication. We will be using Volsync’s restic datamover.</p>
+
+<h3><a href="https://github.com/migtools/volume-snapshot-mover">VOLUMESNAPSHOTMOVER (VSM) CONTROLLER</a>:</h3>
+
+<p dir="auto">The VSM controller is the CSI data movement orchestrator, it is deployed via the OADP Operator once the datamover feature is enabled. This controller has the following responsibilities:</p>
+
+<ul dir="auto">
+	<li>Validates the VolumeSnapshotBackup/VolumeSnapshotRestore Custom Resources.</li>
+	<li>Makes sure that the data movement workflow has the appropriate storage credentials</li>
+	<li>Performs the copy of VolumeSnapshotContent, CSI VolumeSnapshot and PersistentVolumeClaims from application namespace to OADP Operator namespace</li>
+	<li>Triggers the data movement process and subsequently performs the cleanup of extraneous resources created.</li>
+</ul>
+
+<h3><a href="https://github.com/migtools/velero-plugin-for-vsm">VSM PLUGIN</a>:</h3>
+
+<p dir="auto">The upstream Velero plugin for the VolumeSnapshotMover(VSM) controller is to facilitate CSI volumesnapshot data movement from an OpenShift cluster to object storage, and vice versa.</p>
```

Comment:
I would remove `upstream` in this sentence

Reply:
agree